config/solr.yml

```diff
@@ -1,7 +1,7 @@
 # This is a sample config file that points to a solr server for each environment
 development:
-  url: http://localhost:8983/solr/development
+  url: <%= ENV['SOLR_URL'] || 'http://localhost:8983/solr/development' %>
 test:
-  url: <%= "http://127.0.0.1:#{ENV['TEST_JETTY_PORT'] || 8983}/solr/test" %>
+  url: <%= ENV['SOLR_TEST_URL'] || 'http://localhost:8983/solr/test' %>
```

Comment:
Yes exactly. Both have same hosts/ports just different urls. 
`SOLR_URL` = `'http://solr:8983/solr/development`
and `SOLR_TEST_URL` = `'http://solr:8983/solr/test`

I suppose we could have just one as `SOLR_URL` = `http://solr:8983/solr/` and append the development/test to each in this solr.yml.